Project.toml

```diff
@@ -1,18 +1,20 @@
 name = "Legolas"
 uuid = "741b9549-f6ed-4911-9fbf-4a1c0c97f0cd"
 authors = ["Beacon Biosignals, Inc."]
-version = "0.5.16"
+version = "0.6.0"
```

Comment:
Seems like this would constitute a bugfix rather than a breaking release?

Reply:
it introduces new higher compat bounds

Reply:
That’s not considered breaking since Pkg will handle it and not install this version if you aren’t compatible

Reply:
okey dokey

Reply:
```suggestion
version = "0.5.17"
```